.git-blame-ignore-revs

```diff
@@ -0,0 +1,4 @@
+# Reformat with Black
+53a7e3500437a9fd5826bc04758f7116bd7e52dc
+# ClangFormat the C code
```

Comment:
```suggestion
# Format with Black
53a7e3500437a9fd5826bc04758f7116bd7e52dc
# Format the C code with ClangFormat
```

Reply:
I just borrowed the commit messages, but sure :)

Reply:
...my intention was to become closer to the commit messages. https://github.com/python-pillow/Pillow/commit/53a7e3500437a9fd5826bc04758f7116bd7e52dc is "Format with Black" and https://github.com/python-pillow/Pillow/commit/46b7e86bab79450ec0a2866c6c0c679afb659d17 is "Format with ClangFormat"